SpatialGDK/Documentation/content/glossary.md

```diff
@@ -475,20 +480,15 @@ Once you’ve chosen a label for the worker type (for example, myWorkerType), yo
 >
 > [Worker configuration file `worker.json`](https://docs.improbable.io/reference/latest/shared/worker-configuration/worker-configuration)
 
-### Worker types
-
-> First, see [workers](#worker).
-
-There are two generic types of worker that define how you would want to connect these workers and what kind of capabilities they have:
-
-* [server-worker](#server-workers)
-* [client-worker](#client-workers)
-
-Within these broad types, you can define your own worker sub-types to create more specialized workers.
-
 ### Zoning
 
-Zoning is when you split out the computation of the SpatialOS world spatially, so that each [worker instance](#worker-types-and-worker-instances) of a particular type is responsible for computing the objects in a particular area.
+Zoning is one of the multiserver options for working with SpatialOS. It involves splitting up the world into zones, known as “areas of [authority](#authority)”, with a different [server-worker instance](#server-workers) responsible for each. A server-worker instance can make updates only to Actors that are in its area of authority.
```

Comment:
Might be worth adding at the end of the first sentence that the other one is "offloading"?